src/pages/Header/Header.tsx

```diff
@@ -1,39 +1,179 @@
-                Become better !!!
-            </div>
-
-            <nav>
-                <ul className="flex justify-center space-x-6">
-                    <li className="transform hover:scale-110 transition duration-200">
-                        <Link className="text-lg font-semibold text-green-500 hover:text-orange-500" to="/">Home</Link>
-                    </li>
-                    <li className="transform hover:scale-110 transition duration-200">
-                        <Link className="text-lg font-semibold text-green-500 hover:text-orange-500"
-                              to="/new">Discover</Link>
-                    </li>
-                    <li className="transform hover:scale-110 transition duration-200">
-                        <Link className="text-lg font-semibold text-green-500 hover:text-orange-500"
-                              to="/registration">Sign up</Link>
-                    </li>
-                    <li className="transform hover:scale-110 transition duration-200">
-                        <Link className="text-lg font-semibold text-green-500 hover:text-orange-500"
-                              to="/authorization">Login</Link>
-                    </li>
-                </ul>
-            </nav>
-
-        </header>
-    );
-}
+import logo from "../../assets/images/logo.png";
+
+const Header = () => {
+  return (
+    <div className="bg-indigo-900 text-white">
+      <nav className="mx-auto flex flex-col lg:flex-row items-center justify-between p-6 lg:px-8">
+        <div className="lg:w-1/4 w-full flex items-center justify-center">
+          <img
+            src={logo}
+            alt="Healthcare logo"
+            className="mx-auto"
+            style={{ maxWidth: "130px", height: "auto", margin: "0" }}
+          />
+        </div>
+
+        <div className="w-full flex items-center justify-center lg:justify-between">
+          <ul
+            id="main-menu"
+            className="flex flex-col lg:flex-row items-center space-y-6 lg:space-y-0 lg:space-x-6"
+          >
+            <li className="main-menu-item">
+              <a href="/home">Homepage</a>
+            </li>
+            <li className="main-menu-item">
+              <a href="/about">About us</a>
+            </li>
+            <li className="main-menu-item">
+              <a href="/profile">Profile</a>
+            </li>
+            <li className="main-menu-item">
+              <a href="/services">Services</a>
+            </li>
+          </ul>
+        </div>
+
+        <div className="lg:w-1/4 w-full flex items-center justify-center lg:justify-end">
+          <button
+            className="text-base font-semibold h-full leading-8 text-gray-900 bg-F8E8EA rounded-lg py-0 px-4 whitespace-nowrap"
+            style={{
+              backgroundColor: "#F8E8EA",
+              color: "#18172B",
+              borderRadius: "8px",
+            }}
+            onClick={() => (window.location.href = "#")}
```

Comment:
in react it's bad to use (window.location.href = "#" for performence and it's hard to maintain
use
<Link to="#"